context/AuthHelper.tsx

```diff
@@ -59,46 +44,31 @@ export const loginAccount = async ({
 };
 
 /**
- * Get user data from the session
- * @param router - router to navigate the page
- * @param updateUser - updateUser() function
- * @param resetUser - resetUser() function
- * @param setIsSignedIn - changes to false if errors
+ * Log out and clear session state
  * @returns {Promise<void>}
  */
-export const getUser = async (
-  router: NextRouter,
-  updateUser: (id: string, email: string, leagues: any) => void,
-  resetUser: () => void,
-  setIsSignedIn: (value: boolean) => void,
-) => {
-  if (!isSessionInLocalStorage()) {
-    router.push('/login');
-    return;
-  }
-
+export const logoutAccount = async ({
+  resetUser,
+  setIsSignedIn,
+  router,
+}: {
+  resetUser: () => void;
+  setIsSignedIn: (bool: false) => void;
+  router: NextRouter;
+}): Promise<void | Error> => {
   try {
-    const user = await account.get();
-    const userData: IUser = await getCurrentUser(user.$id);
-    updateUser(userData.id, userData.email, userData.leagues);
-    return userData;
-  } catch (error) {
-    resetUser();
+    await account.deleteSession('current');
     setIsSignedIn(false);
+    resetUser(); // Reset user data in the store
+    toast.custom(
+      <Alert variant={AlertVariants.Success} message="Logged Out" />,
```

Comment:
There is some inconsistency with the single quotes and double quotes within this test. Please update this so you're using on or the other.

Reply:
"I could care less if they are single or double quotes. They work the same." - shashi